File: model/permiso/fitosanitario/Datos.java
/**
 * 
 */
package org.sieca.everis.pdcc.model.permiso.fitosanitario;

import java.util.List;

/**
 * @author ytorregl
 *
 */
public class Datos {
	
	private String lugarEmision;
	private String fechaEmision;
	private String exportadorIdTributaria;
	private String exportadorNombre;
	private String exportadorDireccion;
	private String exportadorTelefono;
	private String exportadorEmail;
	private String exportadorTipoDocumento;
	private String importadorIdTributaria;
	private String importadorNombre;
	private String importadorDireccion;
	private String importadorTelefono;
	private String importadorEmail;
	private String importadorTipoDocumento;
	private String paisOrigen;
	private String paisDestino;
	private String puntoSalida;
	private String puntoIngreso;
	private String modalidadTransporte;
	private String empresaTransportista;
	private String identificacionTransporte;
	private String marcharmoOficial;
	private String inspeccionLugar;
	private String inspeccionFecha;
	private double periodoValidez;
	private String nombreInspector;
	private String declaracionAdicional;
	private List<Detalle> detalles;
	private TratamientoAplicado tratamientoAplicado;
	private String estado;
	/**
	 * 
	 */
	public Datos() {
		super();
		// TODO Auto-generated constructor stub
	}
	/**
	 * @param lugarEmision
	 * @param fechaEmision
	 * @param exportadorIdTributaria
	 * @param exportadorNombre
	 * @param exportadorDireccion
	 * @param exportadorTelefono
	 * @param exportadorEmail
	 * @param exportadorTipoDocumento
	 * @param importadorIdTributaria
	 * @param importadorNombre
	 * @param importadorDireccion
	 * @param importadorTelefono
	 * @param importadorEmail
	 * @param importadorTipoDocumento
	 * @param paisOrigen
	 * @param paisDestino
	 * @param puntoSalida
	 * @param puntoIngreso
	 * @param modalidadTransporte
	 * @param empresaTransportista
	 * @param identificacionTransporte
	 * @param marcharmoOficial
	 * @param inspeccionLugar
	 * @param inspeccionFecha
	 * @param periodoValidez
	 * @param nombreInspector
	 * @param declaracionAdicional
	 * @param detalles
	 * @param tratamientoAplicado
	 * @param estado
	 */
	public Datos(String lugarEmision, String fechaEmision, String exportadorIdTributaria, String exportadorNombre,
			String exportadorDireccion, String exportadorTelefono, String exportadorEmail,
			String exportadorTipoDocumento, String importadorIdTributaria, String importadorNombre,
			String importadorDireccion, String importadorTelefono, String importadorEmail,
			String importadorTipoDocumento, String paisOrigen, String paisDestino, String puntoSalida,
			String puntoIngreso, String modalidadTransporte, String empresaTransportista,
			String identificacionTransporte, String marcharmoOficial, String inspeccionLugar, String inspeccionFecha,
			double periodoValidez, String nombreInspector, String declaracionAdicional, List<Detalle> detalles,
			TratamientoAplicado tratamientoAplicado, String estado) {
		super();
		this.lugarEmision = lugarEmision;
		this.fechaEmision = fechaEmision;
		this.exportadorIdTributaria = exportadorIdTributaria;
		this.exportadorNombre = exportadorNombre;
		this.exportadorDireccion = exportadorDireccion;
		this.exportadorTelefono = exportadorTelefono;
		this.exportadorEmail = exportadorEmail;
		this.exportadorTipoDocumento = exportadorTipoDocumento;
		this.importadorIdTributaria = importadorIdTributaria;
		this.importadorNombre = importadorNombre;
		this.importadorDireccion = importadorDireccion;
		this.importadorTelefono = importadorTelefono;
		this.importadorEmail = importadorEmail;
		this.importadorTipoDocumento = importadorTipoDocumento;
		this.paisOrigen = paisOrigen;
		this.paisDestino = paisDestino;
		this.puntoSalida = puntoSalida;
		this.puntoIngreso = puntoIngreso;
		this.modalidadTransporte = modalidadTransporte;
		this.empresaTransportista = empresaTransportista;
		this.identificacionTransporte = identificacionTransporte;
		this.marcharmoOficial = marcharmoOficial;
		this.inspeccionLugar = inspeccionLugar;
		this.inspeccionFecha = inspeccionFecha;
		this.periodoValidez = periodoValidez;
		this.nombreInspector = nombreInspector;
		this.declaracionAdicional = declaracionAdicional;
		this.detalles = detalles;
		this.tratamientoAplicado = tratamientoAplicado;
		this.estado = estado;
	}
	/**
	 * @return the lugarEmision
	 */
	public String getLugarEmision() {
		return lugarEmision;
	}
	/**
	 * @param lugarEmision the lugarEmision to set
	 */
	public void setLugarEmision(String lugarEmision) {
		this.lugarEmision = lugarEmision;
	}
	/**
	 * @return the fechaEmision
	 */
	public String getFechaEmision() {
		return fechaEmision;
	}
	/**
	 * @param fechaEmision the fechaEmision to set
	 */
	public void setFechaEmision(String fechaEmision) {
		this.fechaEmision = fechaEmision;
	}
	/**
	 * @return the exportadorIdTributaria
	 */
	public String getExportadorIdTributaria() {
		return exportadorIdTributaria;
	}
	/**
	 * @param exportadorIdTributaria the exportadorIdTributaria to set
	 */
	public void setExportadorIdTributaria(String exportadorIdTributaria) {
		this.exportadorIdTributaria = exportadorIdTributaria;
	}
	/**
	 * @return the exportadorNombre
	 */
	public String getExportadorNombre() {
		return exportadorNombre;
	}
	/**
	 * @param exportadorNombre the exportadorNombre to set
	 */
	public void setExportadorNombre(String exportadorNombre) {
		this.exportadorNombre = exportadorNombre;
	}
	/**
	 * @return the exportadorDireccion
	 */
	public String getExportadorDireccion() {
		return exportadorDireccion;
	}
	/**
	 * @param exportadorDireccion the exportadorDireccion to set
	 */
	public void setExportadorDireccion(String exportadorDireccion) {
		this.exportadorDireccion = exportadorDireccion;
	}
	/**
	 * @return the exportadorTelefono
	 */
	public String getExportadorTelefono() {
		return exportadorTelefono;
	}
	/**
	 * @param exportadorTelefono the exportadorTelefono to set
	 */
	public void setExportadorTelefono(String exportadorTelefono) {
		this.exportadorTelefono = exportadorTelefono;
	}
	/**
	 * @return the exportadorEmail
	 */
	public String getExportadorEmail() {
		return exportadorEmail;
	}
	/**
	 * @param exportadorEmail the exportadorEmail to set
	 */
	public void setExportadorEmail(String exportadorEmail) {
		this.exportadorEmail = exportadorEmail;
	}
	/**
	 * @return the exportadorTipoDocumento
	 */
	public String getExportadorTipoDocumento() {
		return exportadorTipoDocumento;
	}
	/**
	 * @param exportadorTipoDocumento the exportadorTipoDocumento to set
	 */
	public void setExportadorTipoDocumento(String exportadorTipoDocumento) {
		this.exportadorTipoDocumento = exportadorTipoDocumento;
	}
	/**
	 * @return the importadorIdTributaria
	 */
	public String getImportadorIdTributaria() {
		return importadorIdTributaria;
	}
	/**
	 * @param importadorIdTributaria the importadorIdTributaria to set
	 */
	public void setImportadorIdTributaria(String importadorIdTributaria) {
		this.importadorIdTributaria = importadorIdTributaria;
	}
	/**
	 * @return the importadorNombre
	 */
	public String getImportadorNombre() {
		return importadorNombre;
	}
	/**
	 * @param importadorNombre the importadorNombre to set
	 */
	public void setImportadorNombre(String importadorNombre) {
		this.importadorNombre = importadorNombre;
	}
	/**
	 * @return the importadorDireccion
	 */
	public String getImportadorDireccion() {
		return importadorDireccion;
	}
	/**
	 * @param importadorDireccion the importadorDireccion to set
	 */
	public void setImportadorDireccion(String importadorDireccion) {
		this.importadorDireccion = importadorDireccion;
	}
	/**
	 * @return the importadorTelefono
	 */
	public String getImportadorTelefono() {
		return importadorTelefono;
	}
	/**
	 * @param importadorTelefono the importadorTelefono to set
	 */
	public void setImportadorTelefono(String importadorTelefono) {
		this.importadorTelefono = importadorTelefono;
	}
	/**
	 * @return the importadorEmail
	 */
	public String getImportadorEmail() {
		return importadorEmail;
	}
	/**
	 * @param importadorEmail the importadorEmail to set
	 */
	public void setImportadorEmail(String importadorEmail) {
		this.importadorEmail = importadorEmail;
	}
	/**
	 * @return the importadorTipoDocumento
	 */
	public String getImportadorTipoDocumento() {
		return importadorTipoDocumento;
	}
	/**
	 * @param importadorTipoDocumento the importadorTipoDocumento to set
	 */
	public void setImportadorTipoDocumento(String importadorTipoDocumento) {
		this.importadorTipoDocumento = importadorTipoDocumento;
	}
	/**
	 * @return the paisOrigen
	 */
	public String getPaisOrigen() {
		return paisOrigen;
	}
	/**
	 * @param paisOrigen the paisOrigen to set
	 */
	public void setPaisOrigen(String paisOrigen) {
		this.paisOrigen = paisOrigen;
	}
	/**
	 * @return the paisDestino
	 */
	public String getPaisDestino() {
		return paisDestino;
	}
	/**
	 * @param paisDestino the paisDestino to set
	 */
	public void setPaisDestino(String paisDestino) {
		this.paisDestino = paisDestino;
	}
	/**
	 * @return the puntoSalida
	 */
	public String getPuntoSalida() {
		return puntoSalida;
	}
	/**
	 * @param puntoSalida the puntoSalida to set
	 */
	public void setPuntoSalida(String puntoSalida) {
		this.puntoSalida = puntoSalida;
	}
	/**
	 * @return the puntoIngreso
	 */
	public String getPuntoIngreso() {
		return puntoIngreso;
	}
	/**
	 * @param puntoIngreso the puntoIngreso to set
	 */
	public void setPuntoIngreso(String puntoIngreso) {
		this.puntoIngreso = puntoIngreso;
	}
	/**
	 * @return the modalidadTransporte
	 */
	public String getModalidadTransporte() {
		return modalidadTransporte;
	}
	/**
	 * @param modalidadTransporte the modalidadTransporte to set
	 */
	public void setModalidadTransporte(String modalidadTransporte) {
		this.modalidadTransporte = modalidadTransporte;
	}
	/**
	 * @return the empresaTransportista
	 */
	public String getEmpresaTransportista() {
		return empresaTransportista;
	}
	/**
	 * @param empresaTransportista the empresaTransportista to set
	 */
	public void setEmpresaTransportista(String empresaTransportista) {
		this.empresaTransportista = empresaTransportista;
	}
	/**
	 * @return the identificacionTransporte
	 */
	public String getIdentificacionTransporte() {
		return identificacionTransporte;
	}
	/**
	 * @param identificacionTransporte the identificacionTransporte to set
	 */
	public void setIdentificacionTransporte(String identificacionTransporte) {
		this.identificacionTransporte = identificacionTransporte;
	}
	/**
	 * @return the marcharmoOficial
	 */
	public String getMarcharmoOficial() {
		return marcharmoOficial;
	}
	/**
	 * @param marcharmoOficial the marcharmoOficial to set
	 */
	public void setMarcharmoOficial(String marcharmoOficial) {
		this.marcharmoOficial = marcharmoOficial;
	}
	/**
	 * @return the inspeccionLugar
	 */
	public String getInspeccionLugar() {
		return inspeccionLugar;
	}
	/**
	 * @param inspeccionLugar the inspeccionLugar to set
	 */
	public void setInspeccionLugar(String inspeccionLugar) {
		this.inspeccionLugar = inspeccionLugar;
	}
	/**
	 * @return the inspeccionFecha
	 */
	public String getInspeccionFecha() {
		return inspeccionFecha;
	}
	/**
	 * @param inspeccionFecha the inspeccionFecha to set
	 */
	public void setInspeccionFecha(String inspeccionFecha) {
		this.inspeccionFecha = inspeccionFecha;
	}
	/**
	 * @return the periodoValidez
	 */
	public double getPeriodoValidez() {
		return periodoValidez;
	}
	/**
	 * @param periodoValidez the periodoValidez to set
	 */
	public void setPeriodoValidez(double periodoValidez) {
		this.periodoValidez = periodoValidez;
	}
	/**
	 * @return the nombreInspector
	 */
	public String getNombreInspector() {
		return nombreInspector;
	}
	/**
	 * @param nombreInspector the nombreInspector to set
	 */
	public void setNombreInspector(String nombreInspector) {
		this.nombreInspector = nombreInspector;
	}
	/**
	 * @return the declaracionAdicional
	 */
	public String getDeclaracionAdicional() {
		return declaracionAdicional;
	}
	/**
	 * @param declaracionAdicional the declaracionAdicional to set
	 */
	public void setDeclaracionAdicional(String declaracionAdicional) {
		this.declaracionAdicional = declaracionAdicional;
	}
	/**
	 * @return the detalles
	 */
	public List<Detalle> getDetalles() {
		return detalles;
	}
	/**
	 * @param detalles the detalles to set
	 */
	public void setDetalles(List<Detalle> detalles) {
		this.detalles = detalles;
	}
	/**
	 * @return the tratamientoAplicado
	 */
	public TratamientoAplicado getTratamientoAplicado() {
		return tratamientoAplicado;
	}
	/**
	 * @param tratamientoAplicado the tratamientoAplicado to set
	 */
	public void setTratamientoAplicado(TratamientoAplicado tratamientoAplicado) {
		this.tratamientoAplicado = tratamientoAplicado;
	}
	/**
	 * @return the estado
	 */
	public String getEstado() {
		return estado;
	}
	/**
	 * @param estado the estado to set
	 */
	public void setEstado(String estado) {
		this.estado = estado;
	}
	

	
	

}

File: model/permiso/fitosanitario/Datos.php
/*/
@Yimis Torreglosa Diaz
/*/
<?php class Datos
{
    //VARIABLES
    private $lugarEmision;
    private $fechaEmision;
    private $exportadorIdTributaria;
    private $exportadorNombre;
    private $exportadorDireccion;
    private $exportadorTelefono;
    private $exportadorEmail;
    private $exportadorTipoDocumento;
    private $importadorIdTributaria;
    private $importadorNombre;
    private $importadorDireccion;
    private $importadorTelefono;
    private $importadorEmail;
    private $importadorTipoDocumento;
    private $paisOrigen;
    private $paisDestino;
    private $puntoSalida;
    private $puntoIngreso;
    private $modalidadTransporte;
    private $empresaTransportista;
    private $identificacionTransporte;
    private $marcharmoOficial;
    private $inspeccionLugar;
    private $inspeccionFecha;
    private $periodoValidez;
    private $nombreInspector;
    private $declaracionAdicional;
    private $TratamientoAplicado;
    private $estado;
    private $Detalle = [[]]			 //PENDIENTE

    //CONSTRUCTOR
    public function __construct()
    {
        //
    }

    // METODOS GETTERS y SETTERS

    /**
     * Get the value of lugarEmision
     */

    public function getLugarEmision()
    {
        return $this->lugarEmision;
    }

    /**
     * Set the value of lugarEmision
     *
     * @return  self
     */

    public function setLugarEmision($lugarEmision)
    {
        $this->lugarEmision = $lugarEmision;

        return $this;
    }

    /**
     * Get the value of fechaEmision
     */

    public function getFechaEmision()
    {
        return $this->fechaEmision;
    }

    /**
     * Set the value of fechaEmision
     *
     * @return  self
     */

    public function setFechaEmision($fechaEmision)
    {
        $this->fechaEmision = $fechaEmision;

        return $this;
    }

    /**
     * Get the value of exportadorIdTributaria
     */

    public function getExportadorIdTributaria()
    {
        return $this->exportadorIdTributaria;
    }

    /**
     * Set the value of exportadorIdTributaria
     *
     * @return  self
     */

    public function setExportadorIdTributaria($exportadorIdTributaria)
    {
        $this->exportadorIdTributaria = $exportadorIdTributaria;

        return $this;
    }

    /**
     * Get the value of exportadorNombre
     */

    public function getExportadorNombre()
    {
        return $this->exportadorNombre;
    }

    /**
     * Set the value of exportadorNombre
     *
     * @return  self
     */

    public function setExportadorNombre($exportadorNombre)
    {
        $this->exportadorNombre = $exportadorNombre;

        return $this;
    }

    /**
     * Get the value of exportadorDireccion
     */

    public function getExportadorDireccion()
    {
        return $this->exportadorDireccion;
    }

    /**
     * Set the value of exportadorDireccion
     *
     * @return  self
     */

    public function setExportadorDireccion($exportadorDireccion)
    {
        $this->exportadorDireccion = $exportadorDireccion;

        return $this;
    }

    /**
     * Get the value of exportadorTelefono
     */

    public function getExportadorTelefono()
    {
        return $this->exportadorTelefono;
    }

    /**
     * Set the value of exportadorTelefono
     *
     * @return  self
     */

    public function setExportadorTelefono($exportadorTelefono)
    {
        $this->exportadorTelefono = $exportadorTelefono;

        return $this;
    }

    /**
     * Get the value of exportadorEmail
     */

    public function getExportadorEmail()
    {
        return $this->exportadorEmail;
    }

    /**
     * Set the value of exportadorEmail
     *
     * @return  self
     */

    public function setExportadorEmail($exportadorEmail)
    {
        $this->exportadorEmail = $exportadorEmail;

        return $this;
    }

    /**
     * Get the value of exportadorTipoDocumento
     */

    public function getExportadorTipoDocumento()
    {
        return $this->exportadorTipoDocumento;
    }

    /**
     * Set the value of exportadorTipoDocumento
     *
     * @return  self
     */

    public function setExportadorTipoDocumento($exportadorTipoDocumento)
    {
        $this->exportadorTipoDocumento = $exportadorTipoDocumento;

        return $this;
    }

    /**
     * Get the value of importadorIdTributaria
     */

    public function getImportadorIdTributaria()
    {
        return $this->importadorIdTributaria;
    }

    /**
     * Set the value of importadorIdTributaria
     *
     * @return  self
     */

    public function setImportadorIdTributaria($importadorIdTributaria)
    {
        $this->importadorIdTributaria = $importadorIdTributaria;

        return $this;
    }

    /**
     * Get the value of importadorNombre
     */

    public function getImportadorNombre()
    {
        return $this->importadorNombre;
    }

    /**
     * Set the value of importadorNombre
     *
     * @return  self
     */

    public function setImportadorNombre($importadorNombre)
    {
        $this->importadorNombre = $importadorNombre;

        return $this;
    }

    /**
     * Get the value of importadorDireccion
     */

    public function getImportadorDireccion()
    {
        return $this->importadorDireccion;
    }

    /**
     * Set the value of importadorDireccion
     *
     * @return  self
     */

    public function setImportadorDireccion($importadorDireccion)
    {
        $this->importadorDireccion = $importadorDireccion;

        return $this;
    }

    /**
     * Get the value of importadorTelefono
     */

    public function getImportadorTelefono()
    {
        return $this->importadorTelefono;
    }

    /**
     * Set the value of importadorTelefono
     *
     * @return  self
     */

    public function setImportadorTelefono($importadorTelefono)
    {
        $this->importadorTelefono = $importadorTelefono;

        return $this;
    }

    /**
     * Get the value of importadorEmail
     */

    public function getImportadorEmail()
    {
        return $this->importadorEmail;
    }

    /**
     * Set the value of importadorEmail
     *
     * @return  self
     */

    public function setImportadorEmail($importadorEmail)
    {
        $this->importadorEmail = $importadorEmail;

        return $this;
    }

    /**
     * Get the value of importadorTipoDocumento
     */

    public function getImportadorTipoDocumento()
    {
        return $this->importadorTipoDocumento;
    }

    /**
     * Set the value of importadorTipoDocumento
     *
     * @return  self
     */

    public function setImportadorTipoDocumento($importadorTipoDocumento)
    {
        $this->importadorTipoDocumento = $importadorTipoDocumento;

        return $this;
    }

    /**
     * Get the value of paisOrigen
     */

    public function getPaisOrigen()
    {
        return $this->paisOrigen;
    }

    /**
     * Set the value of paisOrigen
     *
     * @return  self
     */

    public function setPaisOrigen($paisOrigen)
    {
        $this->paisOrigen = $paisOrigen;

        return $this;
    }

    /**
     * Get the value of paisDestino
     */

    public function getPaisDestino()
    {
        return $this->paisDestino;
    }

    /**
     * Set the value of paisDestino
     *
     * @return  self
     */

    public function setPaisDestino($paisDestino)
    {
        $this->paisDestino = $paisDestino;

        return $this;
    }

    /**
     * Get the value of puntoSalida
     */

    public function getPuntoSalida()
    {
        return $this->puntoSalida;
    }

    /**
     * Set the value of puntoSalida
     *
     * @return  self
     */

    public function setPuntoSalida($puntoSalida)
    {
        $this->puntoSalida = $puntoSalida;

        return $this;
    }

    /**
     * Get the value of puntoIngreso
     */

    public function getPuntoIngreso()
    {
        return $this->puntoIngreso;
    }

    /**
     * Set the value of puntoIngreso
     *
     * @return  self
     */

    public function setPuntoIngreso($puntoIngreso)
    {
        $this->puntoIngreso = $puntoIngreso;

        return $this;
    }

    /**
     * Get the value of modalidadTransporte
     */

    public function getModalidadTransporte()
    {
        return $this->modalidadTransporte;
    }

    /**
     * Set the value of modalidadTransporte
     *
     * @return  self
     */

    public function setModalidadTransporte($modalidadTransporte)
    {
        $this->modalidadTransporte = $modalidadTransporte;

        return $this;
    }

    /**
     * Get the value of empresaTransportista
     */

    public function getEmpresaTransportista()
    {
        return $this->empresaTransportista;
    }

    /**
     * Set the value of empresaTransportista
     *
     * @return  self
     */

    public function setEmpresaTransportista($empresaTransportista)
    {
        $this->empresaTransportista = $empresaTransportista;

        return $this;
    }

    /**
     * Get the value of identificacionTransporte
     */

    public function getIdentificacionTransporte()
    {
        return $this->identificacionTransporte;
    }

    /**
     * Set the value of identificacionTransporte
     *
     * @return  self
     */

    public function setIdentificacionTransporte($identificacionTransporte)
    {
        $this->identificacionTransporte = $identificacionTransporte;

        return $this;
    }

    /**
     * Get the value of marcharmoOficial
     */

    public function getMarcharmoOficial()
    {
        return $this->marcharmoOficial;
    }

    /**
     * Set the value of marcharmoOficial
     *
     * @return  self
     */

    public function setMarcharmoOficial($marcharmoOficial)
    {
        $this->marcharmoOficial = $marcharmoOficial;

        return $this;
    }

    /**
     * Get the value of inspeccionLugar
     */

    public function getInspeccionLugar()
    {
        return $this->inspeccionLugar;
    }

    /**
     * Set the value of inspeccionLugar
     *
     * @return  self
     */

    public function setInspeccionLugar($inspeccionLugar)
    {
        $this->inspeccionLugar = $inspeccionLugar;

        return $this;
    }

    /**
     * Get the value of inspeccionFecha
     */

    public function getInspeccionFecha()
    {
        return $this->inspeccionFecha;
    }

    /**
     * Set the value of inspeccionFecha
     *
     * @return  self
     */

    public function setInspeccionFecha($inspeccionFecha)
    {
        $this->inspeccionFecha = $inspeccionFecha;

        return $this;
    }

    /**
     * Get the value of periodoValidez
     */

    public function getPeriodoValidez()
    {
        return $this->periodoValidez;
    }

    /**
     * Set the value of periodoValidez
     *
     * @return  self
     */

    public function setPeriodoValidez($periodoValidez)
    {
        $this->periodoValidez = $periodoValidez;

        return $this;
    }

    /**
     * Get the value of nombreInspector
     */

    public function getNombreInspector()
    {
        return $this->nombreInspector;
    }

    /**
     * Set the value of nombreInspector
     *
     * @return  self
     */

    public function setNombreInspector($nombreInspector)
    {
        $this->nombreInspector = $nombreInspector;

        return $this;
    }

    /**
     * Get the value of declaracionAdicional
     */

    public function getDeclaracionAdicional()
    {
        return $this->declaracionAdicional;
    }

    /**
     * Set the value of declaracionAdicional
     *
     * @return  self
     */

    public function setDeclaracionAdicional($declaracionAdicional)
    {
        $this->declaracionAdicional = $declaracionAdicional;

        return $this;
    }

    /**
     * Get the value of TratamientoAplicado
     */

    public function getTratamientoAplicado()
    {
        return $this->TratamientoAplicado;
    }

    /**
     * Set the value of TratamientoAplicado
     *
     * @return  self
     */

    public function setTratamientoAplicado($TratamientoAplicado)
    {
        $this->TratamientoAplicado = $TratamientoAplicado;

        return $this;
    }

    /**
     * Get the value of estado
     */

    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * Set the value of estado
     *
     * @return  self
     */

    public function setEstado($estado)
    {
        $this->estado = $estado;

        return $this;
    }

    /**
     * Get the value of Detalle
     */

    public function getDetalle()
    {
        return $this->Detalle;
    }

    /**
     * Set the value of Detalle
     *
     * @return  self
     */

    public function setDetalle($Detalle)
    {
        $this->Detalle = $Detalle;

        return $this;
    }
} ?>
